econml/orf/_ortho_forest.py

```diff
@@ -248,6 +248,8 @@ def __init__(self,
         self.verbose = verbose
         self.batch_size = batch_size
         self.categories = categories
+        if self.categories != 'auto':
+            self.categories = [self.categories]  # OneHotEncoder expects a 2D array with features per column
```

Comment:
This and the corresponding changes to `DMLOrthoForest.fit` and `DROrthoForest.fit` look wrong.  In order to better align with SKLearn and user expectations it should be possible to modify categories between calls to fit to use the same estimator with datasets of different shapes.

Reply:
Some of this is because the rebase onto master kept some old code around. I fixed the issues and I implemented a special setter for `categories` in `TreatmentExpansionMixin`, see latest commit.